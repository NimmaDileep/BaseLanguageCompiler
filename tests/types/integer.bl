var x: Int = 10;

print(x);

var y: Int = -212;

print(y);

var z: Int = 0;
print(z);

var w: Int = -0;

print(w);


print(2394);
