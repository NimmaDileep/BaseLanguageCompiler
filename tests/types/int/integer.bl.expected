10
-212
0
0
2394
